src/server/tiered_storage.cc

```diff
@@ -6,6 +6,7 @@
 
 #include <mimalloc.h>
 
+#include <cstddef>
```

Comment:
I really like clangd, but it's includes are so annooooying

Reply:
`-header-insertion=never`